cli/dcoscli/package/main.py

```diff
@@ -276,11 +279,13 @@ def _confirm(prompt, yes):
                     "'{}' is not a valid response.".format(response))
 
 
-def _install(package_name, options_path, app_id, cli, app, yes):
+def _install(package_name, sw_version, options_path, app_id, cli, app, yes):
```

Comment:
`sw_version` conflicts with an existing variable in this function.  Let's avoid that.